### PR TITLE
[WORT-13] Improve Developer Experience

### PR DESCRIPTION
**Pull request by syahshiimi**
Improving team DX is important to increase velocity. These are mostly cosmetic things that are mundane work that needs to be done to ensure consistency across team as we code.

I've installed these dependencies

1. Tailwind CSS Prettier
2. Prettier Formatter
3. Define an `.nvmrc` file to control node versioning
4. Define an `.npmrc` and `engines` object in package.json to encourage use of yarn or pnpm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,21 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+
+Install Node Version Manager (nvm) [instructions](https://github.com/nvm-sh/nvm) and either `yarn` or `pnpm` to get started.
+
+```bash
+nvm use
+
+pnpm install 
+#or
+yarn install
+````
```
Comment (liangyuanruo) [inline README.md:17]: nit: maybe just recommend one of the two package managers for simplicity. our students are still learning the tools of the trade (npm) so additional choices can be a bit of a distraction

**Review by liangyuanruo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (syahshiimi) [reply README.md:17]: Agreed. Let me rephrase the statement to recommend the use of `npm`, which seems to be the universally agreed upon package manager.

I'll update the `engines` object to accept all three package mangers rather than strictly limiting to either `yarn` or `pnpm`.

If the student wishes to explore other package managers, I feel its their prerogative to do so  😄
